cmake/Modules/OpmStaticTargets.cmake

```diff
@@ -7,15 +7,22 @@
 
 # Macros
 function(opm_from_git repo name revision)
+  if(ARGN)
+    set(COMMANDS BUILD_COMMAND make ${ARGN}
```

Comment:
sure, i can do that. but the generator isn't passed on currently, that would have to be added as well in that case.